edx_proctoring/tests/test_api.py

```diff
@@ -498,6 +499,150 @@ def test_remove_allowance_for_user(self):
         remove_allowance_for_user(student_allowance.proctored_exam.id, self.user_id, self.key)
         self.assertEqual(len(ProctoredExamStudentAllowance.objects.filter()), 0)
 
+    def test_add_time_bulk_allowance(self):
+        """
+        Test to add time bulk allowances.
+        """
+        user_list = self.create_batch_users(3)
+        exam_list = []
+        exam_list.extend(
+            (create_exam(
+            course_id=self.course_id,
+            content_id="1st exam",
+            exam_name="1st exam",
+            time_limit_mins=self.default_time_limit,
+            is_practice_exam=False,
+            is_proctored=True
+            ),
+            create_exam(
+            course_id=self.course_id,
+            content_id="2nd exam",
+            exam_name="2nd exam",
+            time_limit_mins=90,
+            is_practice_exam=False,
+            is_proctored=True
+            ))
+        )
+        add_bulk_allowances(exam_list, user_list, 'additional_time', '30')
+        student_allowance = ProctoredExamStudentAllowance.get_allowance_for_user(
+            exam_list[0], user_list[1], self.key
+        )
+        student_allowance_exam2 = ProctoredExamStudentAllowance.get_allowance_for_user(
+            exam_list[1], user_list[1], self.key
+        )
+        all_course_allowances = get_allowances_for_course(self.course_id)
+        self.assertEqual(len(all_course_allowances), 6)
+        self.assertIsNotNone(student_allowance)
+        self.assertEqual(student_allowance.value, '30')
+        self.assertEqual(student_allowance_exam2.value, '30')
+
+    def test_add_multiplier_bulk_allowance(self):
```

Comment:
Please learn about `ddt` library. Search this repo for `@ddt.data` decorators. This help you combine this test case with the test case above.